dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/net/RumRequestInterceptor.kt

```diff
@@ -98,6 +91,41 @@ internal class RumRequestInterceptor : RequestInterceptor {
         return traceId
     }
 
+    private fun handleErrorStatus(method: String, url: String, statusCode: Int) {
+        if (statusCode >= 400) {
+            GlobalRum.get().addError(
+                ERROR_MSG_FORMAT.format(method, url),
+                ORIGIN_NETWORK,
+                null,
+                mapOf(RumAttributes.HTTP_STATUS_CODE to statusCode)
+            )
+        }
+    }
+
+    private fun getResponseAttributes(response: Response): Map<String, Any?> {
+        val statusCode = response.code()
+        val length = getBodyLength(response)
+        return if (length > 0L) {
+            mapOf(
+                RumAttributes.HTTP_STATUS_CODE to statusCode,
+                RumAttributes.NETWORK_BYTES_WRITTEN to length
+            )
+        } else {
+            mapOf(RumAttributes.HTTP_STATUS_CODE to statusCode)
+        }
+    }
+
+    private fun getBodyLength(response: Response): Long {
```

Comment:
We are adding the body length only for request responses ...we are not doing this also for "POST" requests though.

Reply:
That's also the way it's done on web. They only track the response's body length, not the post request length